### PR TITLE
RHMAP-8254 Updated .travis.yml

### PR DESCRIPTION
**Pull request by psturc**
## Motivation
https://issues.jboss.org/browse/RHMAP-8254
https://issues.jboss.org/browse/RHMAP-7976

## Comments
Build performed here: https://travis-ci.org/psturc/hello-push-cloud/builds/141806975

ping @david-martin, @matzew (cannot ping @adamsaleh - not in this repo)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 node_js:
   - "0.10"
   - "4.4.3"
-sudo: false
-before_install:
-  - npm install -g npm@2.13.5
-  - npm install -g grunt-cli
+before_install: 
+  - git clone https://gist.github.com/psturc/8f386fd74d8b34faf113ff14d7638529
+  - cp 8f386fd74d8b34faf113ff14d7638529/build.sh .
```
Comment (matzew) [inline .travis.yml]: Perhaps, we might think about moving this to a CI/test/automation etc repository ?

Comment (psturc) [inline .travis.yml]: Hi @matzew, do we already have this kind of repo?

**Comment by matzew** [comment]
@psturc can you rebase? looks like here are confluicts

**Comment by psturc** [comment]
@matzew Sure, but I'll probably close this one and create a mass PRs for all the templates again, now with using [fh-build](https://www.npmjs.com/package/fh-build)

**Comment by matzew** [comment]
Sounds good! closing... 😄

**Pull request closed without merging**